src/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ if(OV_COMPILER_IS_INTEL_LLVM)
     ov_disable_deprecated_warnings()
 endif()
 
-if(APPLE AND AARCH64)
+if(APPLE AND AARCH64 AND NOT OV_FORCE_ADHOC_SIGN)
```

Comment:
Could you please add the description of this flag (what is it for and when or how it should be used)?